components/back-top/doc/index.en-US.md

```diff
@@ -5,6 +5,8 @@ title: BackTop
 cover: https://gw.alipayobjects.com/zos/alicdn/tJZ5jbTwX/BackTop.svg
 ---
 
+* Note: This component will be removed later. Please use `<nz-float-button-top></nz-float-button-top>` in the FloatButton component.
```

Comment:
Just wondering, does that mean the backtop component is to be considered deprecated?  
Is it in preparation for Ant Design 5?